### DISCUSSION TITLE
Save window/frames position

### DISCUSSION THREAD

**Discussion by janstieler** [coder/code-server#5101] [Q&A]
Hi,
does anybody know who I can save my window/frame positions?
I work on a splittet Screen and the Explorer and the Terminal needs to much space if its open. So I have everytime to resize them.

Chers

**Comment by jsjoeio** [comment]
Hmm...that seems like something you could solve with a window manager. Unless you're talking about saving the window/frame positions inside VS Code? (which I'm not sure is possible)
